scripts/sinhPrefillData.sql

```diff
@@ -0,0 +1,84 @@
+INSERT INTO team 
```

Comment:
should this be committed? If so can we rename it to just `prefillData`

Reply:
agreed